_config.yml

```diff
@@ -0,0 +1,25 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: GeoAPI Documentation
+email: contact@beta.gouv.fr
+description: > #
+ La GéoAPI est une boîte-à-outils
+ facile à prendre en main pour rendre vos applications
+  et bases de données plus intelligentes, en terme de positionnement
+  et de connaissance des territoires.
```

Comment:
Too long. `description` is for SEO, limit it to 160 chars for search engine excerpts.